caps/capabilities_test.go

```diff
@@ -46,9 +46,11 @@ func (s *CapabilitySuite) TestValidateName(c *C) {
 
 func (s *CapabilitySuite) TestAdd(c *C) {
 	repo := NewRepository()
+	err := LoadBuiltInTypes(repo)
```

Comment:
Given the pattern repo := NewRepository(), LoadBuildiInTypes(repo) is repeating I wonder if it would make sense to define a RepoCapsSuite that is doing this boilerplate in the SetUpTests() method and add it to the RepoCapsSuite struct?